src/Microsoft.Health.Dicom.Web/appsettings.json

```diff
@@ -84,7 +84,8 @@
       "EnableFullDicomItemValidation": false,
       "EnableOhifViewer": true,
       "EnableExternalStore": false,
-      "DisableOperations": false
+      "DisableOperations": false,
+      "EnableExternalStoreHealthCheck": true
```

Comment:
This can be false by default, since there is no concept of external store in OSS

Reply:
Once merged in paas, you can make this true by default

Reply:
Good call, makes sense to leave this false here